index.html

```diff
@@ -469,16 +272,69 @@ <h2 class="contact-heading-details color-white">Contact me</h2>
         </div>
       </footer>
     </div>
+    
+
+ <!-- popup session -->
+
+     <!-- popup -->
+    <div class="popupContainer">
+      <div class="popup-wrapper">
```

Comment:
- [ ]  Kindly add a side scroll to the popup modal so the rest of the contents on the page can be accessible 

![image](https://user-images.githubusercontent.com/81932478/168931975-02d9454f-d2e1-49c9-aa1c-79c81cf6b05b.png)

Currently, the page ends where the screenshot above ends and the rest of the modal popup contents are inaccessible.